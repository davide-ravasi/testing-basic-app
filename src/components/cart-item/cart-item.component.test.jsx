import React from "react";
import { shallow } from "enzyme";
import CartItem from './cart-item.component';

describe('CartItem component', () => {
    let wrapper, mockProps;

    beforeEach(() => {
        mockProps = {
            item: {
                imageUrl: 'https://i.ibb.co/bPmVXyP/black-converse.png',
                name: "Black Converse",
                price: 110,
                quantity: 1
            }
        }


        wrapper = shallow(<CartItem  {...mockProps} />);
    })

    it('render basic component', () => {
        expect(wrapper).toMatchSnapshot();
    })

})