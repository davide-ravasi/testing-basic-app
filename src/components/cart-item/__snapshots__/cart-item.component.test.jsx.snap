// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartItem component render basic component 1`] = `
<styled.div>
  <styled.img
    alt="item"
    src="https://i.ibb.co/bPmVXyP/black-converse.png"
  />
  <styled.div>
    <span>
      Black Converse
    </span>
    <span>
      1
       x $
      110
    </span>
  </styled.div>
</styled.div>
`;
